tools/integration_tests/implicitdir/implicitdir_test.go

```diff
@@ -111,6 +112,27 @@ func clearKernelCache() error {
 	return nil
 }
 
+func compareFileContents(t *testing.T, fileName string, filecontent string) {
+
+	err := clearKernelCache()
+	if err != nil {
+		t.Errorf("Clear Kernel Cache: %v", err)
+	}
+	content, err := os.ReadFile(fileName)
```

Comment:
new lines are added to logically seperate code. please add new line after 120 and remove at 122 and add after 125.